client/src/app/components/Toolbar/Toolbar.jsx

```diff
@@ -19,27 +19,73 @@ import { createShoppingList } from 'modules/shopping-list/model/actions';
 import { createCohort } from 'modules/cohort/model/actions';
 import { getCurrentUser } from 'modules/authorization/model/selectors';
 import { UserPropType } from 'common/constants/propTypes';
+import Overlay, { OverlayStyleType } from 'common/components/Overlay';
 
 class Toolbar extends PureComponent {
   state = {
-    cohortFormVisbility: false,
-    shoppingFormVisbility: false
+    cohortFormVisibility: false,
+    formDescription: '',
+    formTitle: '',
```

Comment:
these two should be in form, not here - please try react component methods to reset these values